libpod/runtime_pod_linux.go

```diff
@@ -210,7 +219,7 @@ func (r *Runtime) removePod(ctx context.Context, p *Pod, removeCtrs, force bool)
 
 		// Ensure state appropriate for removal
 		if err := ctr.checkReadyForRemoval(); err != nil {
-			return errors.Wrapf(err, "pod %s has containers that are not ready to be removed", p.ID())
+			return errors.Wrapf(err, "The following container is not ready to be removed from pod %s: %s", p.ID(), ctr.ID())
```

Comment:
This just adds the container ID twice to the error message. The returned error already contains the container ID as seen in the linked issue.